project/Build.scala

```diff
@@ -116,7 +116,10 @@ object Build extends AutoPlugin {
       lazy val compile = "tests/test:compile"
       lazy val fastTest = "tests/fast:test"
       lazy val slowTest = command("tests/slow:test")
-      lazy val scalafixTest = "scalafixTests/test"
+      lazy val scalafixTest = command(
+        "scalafixTests/test",
+        "scalafixOutput/compile"
```

Comment:
@maxov It turned out that we never tested that scalafixOutput compiles, and it didn't 🤦